### PR TITLE
fetch: Adding Siena and Bergamo support

### PR DESCRIPTION
**Pull request by larrydewey**
Both Siena and Bergamo processors are classified under the Genoa family on the AMD Key Distribution Server. This adds the necessary changes to make the requests appropriately for those models.

### CODE REVIEW

File: src/fetch.rs [hunk 1]
```diff
@@ -197,3 +220,23 @@ mod vcek {
...
+    fn test_kds_prod_name_milan_base() {
+        let milan_proc: ProcType = ProcType::Milan;
+        assert_eq!(milan_proc.kds_prod_name(), ProcType::Milan.to_string());
+    }
+
+    #[test]
+    fn test_kds_prod_name_genoa_base() {
+        assert_eq!(ProcType::Genoa.kds_prod_name(), ProcType::Genoa.to_string());
+        assert_eq!(ProcType::Siena.kds_prod_name(), ProcType::Genoa.to_string());
+        assert_eq!(
```
Comment (AdithyaKrishnan) [inline src/fetch.rs:237]: assert_eq!(ProcType::Bergamo.kds_prod_name(), ProcType::Genoa.to_string());

**Review by tylerfanelli — APPROVED** [review]

File: src/fetch.rs (hunk 1 above)
Comment (larrydewey) [reply src/fetch.rs:237]: It wrapped due to linting rules :)

Comment (larrydewey) [reply src/fetch.rs:237]: It wrapped due to linting rules :)

**Review by DGonzalezVillal — APPROVED** [review]
LGTM!

**Pull request merged**